src/functions/parse_cookie_header.php

```diff
@@ -16,7 +16,7 @@
  * overwriting. Thus, the server request should take the cookies from the request header instead.
  *
  * @param string $cookieHeader A string cookie header value.
- * @return array key/value cookie pairs.
+ * @return array<string, string> key/value cookie pairs.
```

Comment:
can we be more strict here? guess its `array<non-empty-string,string>`?
I can't beliefe that a cookie header like this is supported:

```
Cookie: =
```

Reply:
Doesn't seem to be supported: will add `non-empty-string` here, although I cannot safely add an assertion, and will expand the baseline instead

Reply:
dfa990d294c2e6300699978e78c63724647c5d2a